test/TestInternalGrains/PersistenceTestGrains.cs

```diff
@@ -668,9 +668,9 @@ private void CheckRuntimeEnvironment()
             var callStack = new StackTrace();
 
             executing = true;
-            Assert.AreEqual(_scheduler, scheduler, "Wrong TaskScheduler {0} Caller:{1}", scheduler, callStack);
-            Assert.IsNotNull(context, "Null ActivationContext -- Expected: {0} Caller:{1}", _context, callStack);
```

Comment:
Do you mean to preserve the message in code, or have it throw that specific message in the event that the assertion fails?  Either way, the answer is yes, but I want to make sure I do the right thing :smiley: